tests/integration/test_integration_artifactory_path.py

```diff
@@ -173,6 +175,81 @@ def test_deploy_file(path):
     p.unlink()
 
 
+def test_deploy_file_by_checksum(path):
```

Comment:
at the moment we need to put more effort on unittests rather than integration since we do not have Artifactory instance so far

Reply:
I agree with you.
For the api wrapper, it seems there is not a proper solution for it.
Instead, integration tests can cover more cases.

Reply:
@zhan9san,
please pull latest develop. I added multiple **unittests** for `deploy`. Just follow the same concept to add tests to your functions

Reply:
Fixed

Reply:
well done